models/migrations/v85.go

```diff
@@ -58,6 +58,9 @@ func hashAppToken(x *xorm.Engine) error {
 
 		if len(indexes) >= 1 {
 			_, err = sess.Exec("DROP INDEX UQE_access_token_sha1 ON access_token")
+			if err != nil {
```

Comment:
The err will be handled on line 68

Reply:
Same as above.
A new **local** `err` var is created on line 54 in the block.

Reply:
Same as before.